# Data Structure for Image
# sensor_msgs/Image data
std_msgs/UInt8MultiArray data
string gtJson_filepath
string msglabel
int16 sceneid
int16 imgid